invenio_records_permissions/generators.py

```diff
@@ -230,6 +230,26 @@ def query_filter(self, identity=None, **kwargs):
         return reduce(operator.or_, queries)
 
 
+class AllowedByAction(Generator):
+    """Allows admin-enabled actions.
```

Comment:
this looks too naive, do we really need it?

Reply:
We agreed on naming it `AdminAction` generator so it can be used for admin-related activities